src/Pim/Bundle/CatalogBundle/Elasticsearch/Sorter/InGroupSorter.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\Elasticsearch\Sorter;
+
+use Pim\Component\Catalog\Query\Sorter\FieldSorterInterface;
+
+/**
+ * @author    Philippe Mossière <philippe.mossiere@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class InGroupSorter extends BaseSorter implements FieldSorterInterface
+{
+    const GROUPS_KEY = 'groups';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addFieldSorter($field, $direction, $locale = null, $scope = null)
+    {
+        parent::addFieldSorter(self::GROUPS_KEY, $direction, $locale, $scope);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsField($field)
+    {
+        return (strpos($field, 'in_group_') !== false);
```

Comment:
in_group_ ?Where do that come from ? can you explain a bit more how variant_group / group are handled in your PR's description ?